integration-tests/api/__tests__/admin/draft-order/draft-order.js

```diff
@@ -873,5 +873,30 @@ describe("/admin/draft-orders", () => {
       expect(dorder.cart.shipping_address.last_name).toEqual("james")
       expect(dorder.cart.discounts[0].code).toEqual("TEST")
     })
+
+    it("remove customer", async () => {
+      const api = useApi()
+
+      const response = await api.post(
+        "/admin/draft-orders/test-draft-order",
+        {
+          customer_id: "",
```

Comment:
suggestion: If we make the property nullable then I think that null explicitly would be better. We do that in other places where we use null as explicitly setting the prop to null and undefined as doing nothing.